src/dagmc.cpp

```diff
@@ -231,6 +236,11 @@ void load_dagmc_geometry()
     model::overlap_check_count.resize(model::cells.size(), 0);
   }
 
+  if (!graveyard) {
+    warning("No graveyard volume found in the DagMC model.");
+    warning("This may result in lost particles and rapid simulation failure.");
```

Comment:
This will result in two separate warnings. Just have the two strings implicitly joined within one call to `warning()`